controllers/performanceprofile_controller.go

```diff
@@ -1,11 +1,27 @@
-package performanceprofile
+/*
```

Comment:
commenting here because github don't allow me to do so in the next commit: please explain a bit more why the files are not needed anymore: is their function now merged in other files?

Reply:
No, the files you are referring were auto generated by a prev version of operator-sdk. The new version does not generate them anymore. They are doing things differently

Reply:
Ok, so their function is now performed by other code, right?

Reply:
yes, possibly hidden by controller-gen